sys/include/storage/nvs.h

```diff
@@ -0,0 +1,102 @@
+
+/**
+ * Copyright 2020 btcven and Locha Mesh developers
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @ingroup     storage_nvs
```

Comment:
```suggestion
 * @defgroup    storage_nvs
 * @ingroup     storage_nvs
```

There needs to be a group definition when a new one is created, on the .h file.